index.html

```diff
@@ -65,5 +80,294 @@ <h2 class="SupportingText">I'am a Software Developer</h2>
+      </div>
+
+      <div class="tag5">
+        <img src="images/Tag (5).png" alt="Html">
+        <img src="images/Tag (6).png" alt="Bootstrap">
+        <img src="images/Tag (7).png" alt="Ruby">
+      </div>
+
+      <div class="Action3">
+        <a href="#"></a><img src="images/Action.png" alt="actions">
+      </div>
+
+      <div class="Testseeproject2">
+        <a href="#"></a><img src="images/See project 2.png" alt="seeproject">
+      </div> 
+
+      <div class="MaskGroup5"> 
+        <img src="images/Img Placeholder (4).png" alt="MaskGroup">
+      </div>
+
+      <div class="TextTitlePost4">
+        <p>Profesional Art Printing Data</p>
+      </div>
+
+      <div class="textsupportingtext4">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+      </div>
+
+      <div class="tag6">
+        <img src="images/Tag (5).png" alt="Html">
+        <img src="images/Tag (6).png" alt="Bootstrap">
+        <img src="images/Tag (7).png" alt="Ruby">
+      </div>
+
+      <div class="Action4">
+        <a href="#"></a><img src="images/Action.png" alt="actions">
+      </div>
+
+      <div class="Testseeproject3">
+        <a href="#"></a><img src="images/See project 2.png" alt="seeproject">
+      </div> 
+
+      <div class="MaskGroup6"> 
+        <img src="images/Img Placeholder (6).png" alt="MaskGroup">
+      </div>
+
+      <div class="TextTitlepost5">
+        <p>Profesional Art Printing Data</p>
+      </div>
+
+      <div class="textsupportingtext5">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+      </div>
+
+      <div class="tag7">
+        <img src="images/Tag (5).png" alt="Html">
+        <img src="images/Tag (6).png" alt="Bootstrap">
+        <img src="images/Tag (7).png" alt="Ruby">
+      </div>
+
+      <div class="Action5">
+        <a href="#"></a><img src="images/Action.png" alt="actions">
+      </div>
+
+      <div class="Testseeproject4">
+        <a href="#"></a><img src="images/See project 2.png" alt="seeproject">
+      </div> 
+
+    </section>
+
+    <section class="About-me-section">
+      <div class="Primarytext">
+        <p>About me</p>
+      </div>
+
+      <div class="TextDescription">
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      </div>
+
+      <div class="Enable">
+        <img src="images/Enable.png" alt="enable">
+      </div>
+
+      <div class="Grouplogo">
+        <img src="images/Group (1).png" alt="Grouplogo">
+      </div>
+      <div class="Vectorline">
+        <img src="images/Vector 4 (1).png" alt="vector">
+      </div>
+
+      <div class="Language">
+        <img src="images/Language (1).png" alt="Language">
```

Comment:
- I noticed that you added the text below as an image.
![image](https://user-images.githubusercontent.com/102699958/227732488-58a9d8f2-4350-4991-8ce3-f734c04daba0.png)
- [ ] It is should be added as one of the (`hx` tags). eg.(`h2`, `h4`, `h3` ...) etc.